app/notification_test.go

```diff
@@ -861,22 +861,18 @@ func TestGetExplicitMentionsAtHere(t *testing.T) {
 
 	for message, shouldMention := range cases {
 		post := &model.Post{Message: message}
-		if m := getExplicitMentions(post, nil); m.HereMentioned && !shouldMention {
-			t.Fatalf("shouldn't have mentioned @here with \"%v\"", message)
-		} else if !m.HereMentioned && shouldMention {
-			t.Fatalf("should've mentioned @here with \"%v\"", message)
-		}
+		m := getExplicitMentions(post, nil)
+		require.False(t, m.HereMentioned && !shouldMention, "shouldn't have mentioned @here with \"%v\"")
+		require.False(t, !m.HereMentioned && shouldMention, "should've mentioned @here with \"%v\"")
 	}
 
 	// mentioning @here and someone
 	id := model.NewId()
-	if m := getExplicitMentions(&model.Post{Message: "@here @user @potential"}, map[string][]string{"@user": {id}}); !m.HereMentioned {
-		t.Fatal("should've mentioned @here with \"@here @user\"")
-	} else if len(m.Mentions) != 1 || m.Mentions[id] != KeywordMention {
-		t.Fatal("should've mentioned @user with \"@here @user\"")
-	} else if len(m.OtherPotentialMentions) > 1 {
-		t.Fatal("should've potential mentions for @potential")
-	}
+	m := getExplicitMentions(&model.Post{Message: "@here @user @potential"}, map[string][]string{"@user": {id}})
+	require.True(t, m.HereMentioned, "should've mentioned @here with \"@here @user\"")
+	require.Len(t, m.Mentions, 1)
+	require.Equal(t, KeywordMention, m.Mentions[id], "should've mentioned @user with \"@here @user\"")
+	require.LessOrEqual(t, len(m.OtherPotentialMentions), 1, "should've potential mentions for @potential")
```

Comment:
No action necessary, but just a heads-up that we try to use `assert` instead of `require` in any places that won't cause a panic or for anything that's not required for the next step in the test. That way we can run the test once and see multiple failures instead of having to run, fix, run, fix, and so on.